src/CliptoToken.sol

```diff
@@ -42,6 +42,10 @@ contract CliptoToken is ERC721("", ""), ERC721Enumerable, ERC721URIStorage, IERC
 
         nameHash = keccak256(bytes(_name));
         versionHash = keccak256(bytes("0.0.1"));
+
+        /// @notice rate * 10,000, default: 5%
+        royaltyRate = 50_000;
+        scale = 1e5;
```

Comment:
this would be `50_000 / 1e5 = 0.5` which is `50%`
One option is to increase the scale to `1e6`

Reply:
Fixed in latest commit